<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImgTools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgViewerLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
border: 2px solid #000000</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="outputFormatComboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PNG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ICO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPEG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BMP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TIFF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SVG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WEBP</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="imgPathLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>No image Loaded!</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputFormatLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFilePathInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>581</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="outputFilePathLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseOutputPathButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="resizeCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize               X                            Y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeXSpinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeYSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="resetSizeButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate90degButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+90째</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate90NegDegButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-90째</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate180degButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>180째</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="imageRotationSpinBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>63</width>
      <height>25</height>
     </rect>
    </property>
    <property name="suffix">
     <string>째</string>
    </property>
    <property name="maximum">
     <double>359.990000000000009</double>
    </property>
   </widget>
   <widget class="QPushButton" name="resetRotationButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
    </property>
   </widget>
   <widget class="QComboBox" name="imagesComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenFolder"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBackground_Remover"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Open Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionBackground_Remover">
   <property name="text">
    <string>Background Remover</string>
   </property>
   <property name="toolTip">
    <string>Remove the background from an image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
